src/test/kotlin/it/pagopa/wallet/scheduler/services/WalletServiceTest.kt

```diff
@@ -0,0 +1,96 @@
+package it.pagopa.wallet.scheduler.services
+
+import it.pagopa.wallet.scheduler.WalletTestUtils
+import it.pagopa.wallet.scheduler.config.WalletSearchConfig
+import it.pagopa.wallet.scheduler.exceptions.WalletInvalidRangeException
+import it.pagopa.wallet.scheduler.repositories.WalletRepository
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import kotlin.test.assertEquals
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.*
+import org.springframework.data.domain.Limit
+import reactor.core.publisher.Flux
+import reactor.test.StepVerifier
+
+class WalletServiceTest {
```

Comment:
Can we add a test that ensures our query is effectively limited to the first n documents?

Reply:
Done here src/test/kotlin/it/pagopa/wallet/scheduler/repositories/WalletRepositoryTest.kt